Apps/Examples/Examples/All Examples/Annotations/AddMarkersSymbolExample.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+import MapboxMaps
+
+@objc(AddMarkersSymbolExample)
+final class AddMarkersSymbolExample: UIViewController, ExampleProtocol {
+    private enum Constants {
+        static let ICON_KEY = "icon_key"
+        static let BLUE_MARKER_PROPERTY = "icon_blue_property"
+        static let RED_MARKER_PROPERTY = "icon_red_property"
+        static let BLUE_ICON_ID = "blue"
+        static let RED_ICON_ID = "red"
+        static let SOURCE_ID = "source_id"
+        static let LAYER_ID = "layer_id"
+    }
+
+    private var mapView: MapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 55.70651, longitude: 12.554729)
+        let options = MapInitOptions(cameraOptions: CameraOptions(center: centerCoordinate, zoom: 8))
+
+        mapView = MapView(frame: view.bounds, mapInitOptions: options)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+
+        mapView.mapboxMap.onNext(.mapLoaded) { [weak self] _ in
+            guard let self = self else { return }
+            self.finish()
+        }
+
+        mapView.mapboxMap.onEvery(.styleLoaded) { [weak self] _ in
+            self?.prepareStyle()
+        }
+
+        mapView.mapboxMap.style.uri = .streets
+    }
+
+    // MARK: - Style management
+
+    private func prepareStyle() {
+        let style = mapView.mapboxMap.style
+        try? style.addImage(UIImage(named: "blue_marker_view")!, id: Constants.BLUE_ICON_ID, stretchX: [], stretchY: [])
+        try? style.addImage(UIImage(named: "red_marker")!, id: Constants.RED_ICON_ID, stretchX: [], stretchY: [])
+
+        var features = [Feature]()
+        var feature = Feature(geometry: Geometry.point(Point(LocationCoordinate2D(latitude: 55.608166, longitude: 12.65147))))
```

Comment:
```suggestion
        var feature = Feature(geometry: Point(LocationCoordinate2D(latitude: 55.608166, longitude: 12.65147)))
```
Does this work?

Reply:
Yeah, and definitely looks better than. I changed this example and also weeded out other instances of this syntax in a separate commit.